CNV-25574: Release note: NEW - UI - Ability to make a hot-plug disk part of the VM (#58016)
openshift/openshift-docs: virt/virt-4-13-release-notes.adoc

//CNV-25427 Release note: NEW Allow VM admin to expose SSH service using LB in the UI

//CNV-25574 Release note: NEW UI - Ability to make a hot-plug disk part of the VM

> * The option to make a hot-plug volume a persistent volume is added to the xref:../virt/virt-web-console-overview.html#ui-virtualmachine-details-disks_virt-web-console-overview[*Disks* tab].

Reviewer: This should be .adoc rather than .html, right?

> * The option to make a hot-plug volume a persistent volume has been added to the xref:../virt/virt-web-console-overview.html#ui-virtualmachine-details-disks_virt-web-console-overview[*Disks* tab].

Reviewer 2: ```suggestion
* The option to make a hot-plug volume a persistent volume is added to the xref:../virt/virt-web-console-overview.html#ui-virtualmachine-details-disks_virt-web-console-overview[*Disks* tab].
```
Use simple present tense in most cases. See https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense.

Author: Fixed. Thanks!
